docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1917 @@
+{
```

Comment:
<p>maybe a note on that "MT-" is for human and "Mt-" for mice? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>